tutorials/nnvm/from_darknet.py

```diff
@@ -105,24 +106,45 @@
 
 m.run()
 # get outputs
-out_shape = (net.outputs,)
-tvm_out = m.get_output(0).asnumpy().flatten()
+tvm_out = []
+if MODEL_NAME == 'yolov2':
+    layer_out = {}
+    layer_out['type'] = 'Region'
+    # Get the region layer attributes (n, out_c, out_h, out_w, classes, coords, background)
+    layer_attr = m.get_output(1, tvm.nd.empty((7,), dtype='int')).asnumpy()
```

Comment:
You can avoid passing output shape from now onwards.

Reply:
Done